src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/GoogleComputeDAO.scala

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.dsde.workbench.leonardo.dao.google
```

Comment:
Should we move this to workbench libs?

Reply:
Good question -- my thinking was to leave it in Leo alongside `DataprocDAO` until there's a need for it in wb-libs.